app_guides/php/getting_started.adoc

```diff
@@ -0,0 +1,297 @@
+concepts. You do not have to use our sample application, but for the first steps
+it might be helpful.
+
+[[php-git-repo-hosting]]
+=== Git Repository Hosting
+
+To create new application, you must have your source code hosted somewhere on
+the Internet and have it accessible using Git. Popular choices are
+link:https://github.com/[GitHub], link:https://gitlab.com/[GitLab] or
+link:https://bitbucket.org/[BitBucket].
+
+For example, our sample application is hosted at GitHub. For this tutorial, you
+may use the code from the provided repository or you can
+link:https://help.github.com/articles/fork-a-repo/[_fork_] the repository.
+Forking is necessary if you want to follow also through xref:php-gs-step3[Step
+3].
+
+[[php-getting-started-step2]]
+== Step 2: Creating a New Application
+
+[[php-gs-creating-new-app-cli]]
+=== Creating a New Application with the CLI
+
+Open a terminal and at the command prompt enter the following command to create
+a PHP application, replacing the user name in the URL if you have forked, or the
+whole URL if you are using your own application:
+
+----
+$ oc new-app --name=myapp php~https://github.com/OpenShiftDemos/os-sample-php.git
+----
+
+The output should look something like this:
+
+====
+----
+--> Found image bbfc4eb (5 weeks old) in image stream "php" in project "openshift" under tag "5.6" for "php"
+
+    Apache 2.4 with PHP 5.6
+    -----------------------
+    Platform for building and running PHP 5.6 applications
+
+    Tags: builder, php, php56, rh-php56
+
+    * A source build using source code from https://github.com/OpenShiftDemos/os-sample-php.git will be created
+      * The resulting image will be pushed to image stream "myapp:latest"
+    * This image will be deployed in deployment config "myapp"
+    * Port 8080/tcp will be load balanced by service "myapp"
+      * Other containers can access this service through the hostname "myapp"
+
+--> Creating resources with label app=myapp ...
+    imagestream "myapp" created
+    buildconfig "myapp" created
+    deploymentconfig "myapp" created
+    service "myapp" created
+--> Success
+    Build scheduled, use 'oc logs -f bc/myapp' to track its progress.
+    Run 'oc status' to view your app.
+----
+====
+
+The `--name=myapp` option names the application. By default, it would be the
+base name of the URL without an extension, in our case `os-sample-php`. But it
+is much nicer to have the application named `myapp`, which is what we have done
+using this switch.
+
+OpenShift automatically linked php with `PHP 5.6` as the latest version. You
+could as well use `php55` as the template to deploy using the specific version,
+e.g.
+
+[source]
+----
+oc new-app --name=myapp php55~https://github.com/OpenShiftDemos/os-sample-php.git
+----
+
+{product-title} prints a lot of information here that is not that important for
+us at this moment. What we care about is if our application is accessible from
+the Internet and if we can see it running.
```

Comment:
Would this be better phrased as simply "Verify that the application is running by checking the web console.", if that's descriptive enough.